pontoon/base/urls.py

```diff
@@ -48,6 +48,11 @@
         views.download_translation_memory,
         name='pontoon.download_tmx'),
 
+    # Locale-agnostic links
```

Comment:
This URL pattern needs to be moved two blocks up, above the `pontoon.translate` URL pattern, otherwise the language agnosti URL like localhost:8000/projects/pontoon-intro/messages.properties/ gets caught by `pontoon.translate`.